packages/salesforcedx-utils-vscode/src/cli/forceConfigGet.ts

```diff
@@ -8,7 +8,10 @@
 import { SfdxCommandBuilder } from './commandBuilder';
 import { CliCommandExecutor } from './commandExecutor';
 import { CommandOutput } from './commandOutput';
-
+/**
+ * @deprecated
```

Comment:
If this is deprecated is there a reason why we can't just remove it? We could always retrieve the logic from the history if we really need it

Reply:
Great question, we can't remove it right now cause it's being used in the ISV debugger (mainly). We need to refactor some of that code before we can delete this.